### ISSUE TITLE
The `from_` attribute of the spinbox widget doesn't seem to be respected

### ISSUE THREAD

**Issue by zertrin** [tarball69/tkRAD#12]
Hi,

I think that the `from_` attribute of the spinbox XML widget doesn't work.

I have been able to reproduce the problem with the following "minimal example": I took the simple calculator from the Tutorial, changed the `entry` widget in a `spinbox` and added the `from_="3"` and `to="7"` attributes. (see https://gist.github.com/zertrin/3f86faeac839a2dd37c6/revisions ).

The resulting spinbox begins at `0` and ends at `7`. The expected limit is between `3` and `7`.

---
_PS: thanks a lot for this awesome library. With it I have been able to develop a not so simple GUI with no prior knowledge in tkinter and/or other GUI Programming in Python in less than two days. (I must add, that reading the tkinter official documentation was mandatory, because the current documentation of tkRAD does not help much past the few simple examples presented, but this is a topic for another issue that I may submit soon (suggestion for improvements of tkRAD documentation))_

**Comment by tarball69** [comment]
Hi zertrin,
Effectively, I tried the following code sample and I got the same troubleshooting:

```python
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import tkRAD

xml_source = """
    <tkwidget>
        <spinbox
            from_="3"
            to="7"
            layout="pack"
            layout_options="pady=20"
        />
        <ttkbutton
            text="Quit"
            command="@quit"
            layout="pack"
        />
    </tkwidget>
"""

tkRAD.RADApplication().run(xml=xml_source)
```
The problem comes from Tkinter which does *NOT* use the `from_` attribute **internally** for spinboxes. Tkinter uses `from` (without underscore) instead.
You can see this with the following test code sample:
```python
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import tkinter as tk

root = tk.Tk()

spin = tk.Spinbox(root, from_="3", to="7")

attrs = spin.configure()

print("attribute 'from_' in spinbox?", "from_" in attrs)
print("attribute 'from' in spinbox?", "from" in attrs)
print("attribute 'from' contents:", attrs["from"])

spin.pack(padx=20, pady=20)

root.mainloop()
```
**You can easily fix this up** by editing `tkRAD/xml/rad_xml_widget.py` file, then **go to line 2000** in this file, you should get the following method:
```python
    def _parse_attr_from_ (self, attribute, **kw):
        r"""
            starting point float attribute;
            no return value (void);
        """
        # parsed attribute inits
        self._tkRAD_float_support(attribute, **kw)
    # end def
```
just add one line (`attribute.name = "from"`) this way:
```python
    def _parse_attr_from_ (self, attribute, **kw):
        r"""
            starting point float attribute;
            no return value (void);
        """
        # parsed attribute inits
        attribute.name = "from"
        self._tkRAD_float_support(attribute, **kw)
    # end def
```
Afterwhat the spinbox' `from_` XML attribute should work properly.
I'm going to fix this up officially, but here you get the hint to patch it out quickly.
Thank you for your relevant feedback and -of course- thank you for using `tkRAD` library in your own softwares! :+1: 
Regards.

**Comment by tarball69** [comment]
Released v1.6.6 - Core and Spin which officially fixes this bug.
Closing thread.
